src/handlers/gestureHandlerCommon.ts

```diff
@@ -180,3 +181,17 @@ export function findNodeHandle(
   if (Platform.OS === 'web') return node;
   return findNodeHandleRN(node);
 }
+
+let scheduledHandlersQueueFlush:
```

Comment:
Is there anything we can do to explicitely state that this variable holds the `id` returned from `requestAnimationFrame`? The name of this variable is very similar to the name of the method and I'm not sure if I like this or not 😄

Reply:
[Yes, there is](https://github.com/software-mansion/react-native-gesture-handler/pull/1979/commits/da1592520ebf7f94e9a060a68b5bbe83efc6d2a1#diff-88a2920cc09e756b9842ecad2212fb5750c715ffd5eb6f3deea58f0c3bab788aR185).